addon/components/frost-object-browser.js

```diff
@@ -22,6 +22,9 @@ export default Ember.Component.extend({
   classNames: ['frost-object-browser'],
   contentHeight: 505,
   detailLevel: 'low',
+  isHighDetailVisible: true,
+  isLowDetailVisible: true,
+  isMediumDetailVisible: false,
```

Comment:
I guess that's fine. But couldn't you just as easily not pass in a medium until you fix it? It seems like having separate flags for visibility when we already pass in details w.r.t. the LOD is redundant and unnecessary unless there's a use-case for providing the schema for the LOD and not displaying it, which doesn't seem likely.